pilot/pkg/networking/plugin/authn/authentication_test.go

```diff
@@ -851,3 +851,41 @@ func TestBuildSidecarListenerTLSContex(t *testing.T) {
 		}
 	}
 }
+
+func TestGetJwksURI(t *testing.T) {
+	cases := []struct {
+		in                   *authn.Jwt
+		expectedJwksURI      string
+		expectedErrorMessage string
+	}{
```

Comment:
should have test cases for error case too.

Reply:
updated